src/adapter/src/coord/sequencer/inner/create_materialized_view.rs

```diff
@@ -529,19 +529,30 @@ impl Coordinator {
     ) -> Result<StageResult<Box<CreateMaterializedViewStage>>, AdapterError> {
         // Timestamp selection
         let id_bundle = dataflow_import_id_bundle(global_lir_plan.df_desc(), cluster_id);
-        let (as_of, until) = self.select_timestamps(id_bundle, refresh_schedule.as_ref())?;
+        let (dataflow_as_of, storage_as_of, until) =
+            self.select_timestamps(id_bundle, refresh_schedule.as_ref())?;
+        if dataflow_as_of != storage_as_of {
+            tracing::info!(
+                dataflow_as_of = ?dataflow_as_of,
+                storage_as_of = ?storage_as_of,
+                until = ?until,
+                "materialized view timestamp selection",
+            );
```

Comment:
I think this log message should contain some way to identify the MV in question. Also, how bad would it be to log it unconditionally? The selected as-ofs might be interesting even when they differ.

Reply:
> how bad would it be to log it unconditionally?

Ok, I've made it unconditional.

> I think this log message should contain some way to identify the MV in question.

Actually, it seems somebody did some magic to automatically add some context to these log lines. For example, it looks like this now:
`
environmentd: 2024-03-06T16:53:26.436078Z  INFO message_command:coord::handle_execute{session="192604cc-0811-4bad-afe3-5a7321885660"}:coord::handle_execute_inner{stmt="CREATE MATERIALIZED VIEW mv9 WITH (REFRESH = AT mz_now()::text::int8 + '<REDACTED>') AS SELECT x * x FROM t3"}:sequence_create_materialized_view:create_materialized_view_finish: mz_adapter::coord::sequencer::inner::create_materialized_view: materialized view timestamp selection dataflow_as_of=Antichain { elements: [1709744006102] } storage_as_of=Antichain { elements: [1709745006102] } until=Antichain { elements: [1709745006103] }
`

Reply:
Ah, TIL!

I'm not sure how much we want to rely on the tracing context having identifying information in it, and it would be nice to also have the `GlobalId` somewhere, but I assume that can be reconstructed from the name and other surrounding log lines. Fine to leave it as is.